Mergin/plugin.py

```diff
@@ -353,6 +353,12 @@ def view_local_changes(self):
             iface.messageBar().pushMessage("Mergin", "Current project is not a Mergin project.", Qgis.Warning)
             return
 
+        if not unsaved_project_check(force_saving=True):
+            iface.messageBar().pushMessage(
+                "Mergin", "Project contains unsaved modifications, can not compute local changes", Qgis.Warning
```

Comment:
I would extend message a bit, e.g. "Project contains unsaved modifications, can not compute local changes. Save unsaved edits and try again".

Reply:
Done